src/main/java/com/blackducksoftware/integration/hub/alert/web/actions/SlackConfigActions.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.alert.web.actions;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.blackducksoftware.integration.hub.alert.datasource.entity.SlackConfigEntity;
+import com.blackducksoftware.integration.hub.alert.datasource.repository.SlackRepository;
+import com.blackducksoftware.integration.hub.alert.web.ObjectTransformer;
+import com.blackducksoftware.integration.hub.alert.web.model.SlackConfigRestModel;
+
+@Component
+public class SlackConfigActions extends ConfigActions<SlackConfigEntity, SlackConfigRestModel> {
+
+    @Autowired
+    public SlackConfigActions(final SlackRepository slackRepository, final ObjectTransformer objectTransformer) {
+        super(SlackConfigEntity.class, SlackConfigRestModel.class, slackRepository, objectTransformer);
+    }
+
+    @Override
+    public Map<String, String> validateConfig(final SlackConfigRestModel restModel) {
+        return Collections.EMPTY_MAP;
```

Comment:
We should use Collections.emptyMap(); I know it seems trivial, but this way, backward compatibility can't be broken.